sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -1,23 +1,31 @@
 <template>
-  <student-page-container>
+  <student-page-container
+    container-class="student-start-new-application"
```

Comment:
do we have a property called "container-class" in "student-page-container"
 i

Reply:
No, it doesn't. I forgot to remove that. Thanks!